### ISSUE TITLE
Overlay totals on graphs

### ISSUE THREAD

**Issue by ngoonee** [beancount/fava#490]
Currently mouseover on graph columns brings up a tooltip showing totals.

Is it possible to overlay such totals on the graph? Currently the only way to view that information (without looking at the tables below) on mobile devices is to press on the graph. Which would be fine except that immediately loads a different page, so it's not so useful for comparing month to month (for example).

**Comment by aumayr** [comment]
IMHO clicking the charts should go to the link on the second click, as does on Github for example. This way, the first click (or better: tap) would show the overlay/tooltip, and the second one follow the link.

**Comment by ngoonee** [comment]
That would be fine if possible.

**Comment by xentac** [comment]
I agree that this would be way helpful.  In the meantime, if you drag on the graph near where you want to see, the popup should show up.  At least it does in Chrome when I use it on Android.

**Comment by yagebu** [comment]
These [Google Material design guidelines](https://material.io/guidelines/components/tooltips.html) suggest that a tooltip should show on "tap and hold" on mobile. That's also what I usually do on mobile when on a desktop I would hover. This philosophy seems to work well for buttons (where clicking is the primary behaviour) but doesn't seem to apply so well to most areas of our charts where hovering for the tooltip is the more frequent behaviour. I still think that for areas in the charts that are primarily clickable we could with the guidelines.

It seems nigh-impossible and messy to do stuff like distinguishing between taps/clicks/double-taps/etc in JS. So maybe we can individually adjust the charts to be better suited for touch interfaces. There's only three charts responding to clicks AFAIK:

- Bar Charts: Actually on my laptop as well, I've found these to be a bit too clicky, I've more often than I wanted ended up clicking a bar. I think we could move the "click to time filter" to only occur when actually clicking on the time axis. This would allow tapping on the rest of the chart to show the tooltip.
- Treemap Charts: Does anyone use/need the click-to-zoom behaviour? (I never use it) We could just remove it and then one could tap most of the area to show the tooltip (as only the account names are links).
- Sunburst Charts: I think for these requiring tap-and-hold to show the account balance is acceptable.

> IMHO clicking the charts should go to the link on the second click, as does on Github for example. 

@aumayr: Which chart behaves likes this? I tried the "Network" chart and there a single click takes me straight to the commit. I didn't find links in any of the other charts under the "Graphs" tab.
